deploy/deploy.go

```diff
@@ -222,6 +223,56 @@ type KindSpec struct {
 	AdditionalManifests      []string          `yaml:"additionalManifests"`
 }
 
+var (
+	vOut                  = bytes.NewBuffer([]byte{})
+	vExec execerInterface = kexec.NewExecer(vOut, vOut)
+)
+
+type version struct {
+	Major int
```

Comment:
Do these need to be exported?

Reply:
i am just signally to users that those 3 fields are usable